src/main/resources/db/migration/20240523172055399__add_unit_of_orderable_uuid_to_stock_event_line_items.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE stock_event_line_items ADD COLUMN unitoforderableid UUID;
```

Comment:
Update [schema](https://github.com/OpenLMIS/openlmis-stockmanagement/tree/master/src/main/resources/schemas) for that entity.

Reply:
updated